autoload/instrument_input/input_instrument.gd

```diff
@@ -1,19 +1,17 @@
 class_name InputInstrument
 extends Node
 
-
 signal note_started(frequency)
 signal note_ended(frequency)
 
-signal activated()
-signal deactivated()
-
+signal activated
+signal deactivated
```

Comment:
Any specific reason for this change? I like showing that there are no arguments by using empty parentheses.